src/styles/components/_code-block.scss

```diff
@@ -15,7 +15,7 @@ $breakpoint-tablet: 820px;
       background-image: url("../../../public/images/cmt_inventory_arrow.svg");
       transform: rotate(-30deg);
       position: absolute;
-      left: 11rem;
+      left: 25rem;
```

Comment:
Setting `left` here won't necessarily do what we want it to do, especially with different string lengths. We can modify our `transform` property, though, so that the image is always in the right spot relative to the text:

```suggestion
      transform: translateX(0.5rem) rotate(-30deg);
      position: absolute;
```

Reply:
I was wondering how to do something like this myself!

Reply:
![image](https://user-images.githubusercontent.com/49820744/200863631-ee3dce4c-4059-4b85-ba7e-18272f4b4435.png)
🏆

Reply:
I love learning stuff this way! :) :)